docs/DeveloperGuide.md

```diff
@@ -293,23 +293,35 @@ The following activity diagram summarizes what happens when a user enters an `ed
 ### MarkRoomsUnclean Command
 
 #### Implementation
-* The `markRoomsUnclean` command edits all the guests in GuestBook and changes their isRoomClean statuses to "no". It takes in no additional inputs or fields.
+* The `markroomsunclean` command edits all the guests in GuestBook and changes their isRoomClean statuses to "no". It takes in no additional inputs or fields.
 
-The following activity diagram summarizes what happens when a user enters a `markRoomsUnclean` command.
+The following activity diagram summarises what happens when a user enters a `markroomsunclean` command.
 
 ![MarkRoomsUncleanActivityDiagram](images/MarkRoomsUncleanActivityDiagram.png)
 
 #### Design Considerations:
 **Aspect: The scope at which the command changes all guests' isRoomClean statuses**
-* Alternative 1: Allow `markRoomsUnclean` command to operate only on the last shown list instead of the entire list. This is to Standardise how edits are made across the commands (e.g. edit and delete).
+* Alternative 1: Allow `markroomsunclean` command to operate only on the last shown list instead of the entire list. This is to Standardise how edits are made across the commands (e.g. edit and delete).
```

Comment:
Is Standardise supposed to be capitalised here?

Reply:
Yes